packages/core/src/animation/AnimationEvent.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * AnimationEvent lets you call a script function similar to SendMessage as part of playing back an animation.
+ */
+export class AnimationEvent {
+  /**
+   * The time when the event be triggered.
+   */
+  time: number;
```

Comment:
simple field comments write in one line.

Reply:
add a public field `functionName: string`

Reply:
done